neptune/internal/init_impl.py

```diff
@@ -61,7 +62,23 @@ def init(
         raise ValueError('connection_mode should be on of ["async", "sync", "offline"]')
 
     project_obj = backend.get_project(project)
-    exp = backend.create_experiment(project_obj.uuid)
+
+    notebook_id = None
+    notebook_path = None
+    checkpoint_id = None
+
+    if os.getenv(NOTEBOOK_ID_ENV_NAME, None) is not None:
+        notebook_id = uuid.UUID(os.environ[NOTEBOOK_ID_ENV_NAME])
+    if os.getenv(NOTEBOOK_PATH_ENV_NAME, None) is not None:
+        notebook_path = os.environ[NOTEBOOK_PATH_ENV_NAME]
+    if notebook_id is not None and notebook_path is not None:
+        checkpoint_id = Notebook(backend, notebook_id, notebook_path).create_checkpoint()
```

Comment:
Check for None.

Reply:
I don't need - I pass this downstream to experiment creation where it's sent to backend as None